docs/test-plan.md

```diff
@@ -143,6 +143,39 @@ Run one of the above MRVAs, but cancel it from within VS Code:
 - Check that the workflow run is also canceled.
 - Check that any available results are visible in VS Code.
 
+### CodeQL Model Editor
+
+Note: this is still behind feature flags.
+
+#### Test Case 1: Opening the model editor
+
+1. Download the `dsp-testing/sql2o-example` database from GitHub.
```

Comment:
Is there another repo we can use which is public? We should try to avoid using internal-only repositories for this public test plan.

Reply:
Ah, doh. I cannot seem to get used to the public repo :disappointed: